parlai/agents/seq2seq/seq2seq.py

```diff
@@ -248,23 +249,15 @@ def __init__(self, opt, shared=None):
                     embs = vocab.GloVe(
                         name='840B',
                         dim=300,
-                        cache=os.path.join(
```

Comment:
just checking: is this the same as 
https://github.com/facebookresearch/ParlAI/blob/master/parlai/zoo/glove_vectors/build.py ?


    opt  = { 'datapath': datapath }
    fnames = ['glove.840B.300d.zip']
    download_models(opt, fnames, 'glove_vectors', use_model_type=False,
                    path = "http://nlp.stanford.edu/data")

not clear it is..

Reply:
we should probably just remove parlai/zoo/glove_vectors right? torchtext has its own code for downloading its vectors.

Reply:
it is being used by drqa, if you can make drqa work with the other then yes! fine..! be good to get drqa to work with fasttext as well, anyway..

Reply:
but yes it is the same, download_models uses `os.path.join(opt['datapath'], 'models', model_folder)` where model_folder here is the `glove_vectors` string in that call

Reply:
i thought one might be a binary file and one a text file or something? i guess just check drqa still works please